content/en/logs/explorer/advanced_filtering.md

```diff
@@ -0,0 +1,89 @@
+
+## Filter logs with subqueries
+
+Use subqueries when you want to filter the results of a query based on the results of a secondary query. See [subquery examples](#subquery-examples) for two example scenarios.
+
+To add a subquery filter:
+
+1. Navigate to [Log Explorer][1].
+1. Enter a query in the search bar to filter your logs. This is the main query.
+1. Click **+ Add**.
+1. Under the **Add Query Filter** section, select **Logs**.
+
+This introduces new elements to the query editor:
+
+{{< img src="logs/explorer/advanced_filtering/subquery.png" alt="The subquery editor under the main search query" style="width:95%;" >}}
+
+1. In the **where** field, use the dropdown menu to select the attribute you want to correlate on. The available attributes are from the logs returned by your main query.
+1. Define your subquery filter in the **from** field.
+1. In the **Select Column** dropdown menu that precedes the subquery filter, select the attribute you want to group and sort your subquery results by.  
+1. Select whether you want to use the **IN** or **NOT IN** operator:  
+    - The results from the **IN** operator only include logs where the attribute value is also found in the subquery results. For example, if you only want to see logs generated by `service:a` and is associated to a user who is also one of the top users of `service:b`.  
+    - The results from the **NOT IN** operator exclude logs where the attribute value is found in the subquery results. For example, if you only want to see `status:error` logs, but also want to filter out `status:error` logs where users associated with those logs eventually end up with a `status:success` log. See [Filter outdated or superseded logs](#filter-outdated-or-superseded-logs) for a detailed example.  
+1. Optionally, reduce the number of subquery attribute values to match against. The default value and maximum value is `1000`. Choose between **top** (highest frequency values) or **bottom** (lowest frequency values).
+
+### Subquery examples
+
+The following are scenarios where you need to use subqueries to get the information you need from your logs.
+
+#### Filter outdated or superseded logs
+
+Assume you operate an e-commerce platform. A log is generated each time one of your customers tries to place an order. You want to analyze your logs to understand the total value of potential purchases lost due to an ongoing issue with your website.
+
+You realize, however, that an order can fail multiple times before successfully completing: meaning for that particular order ID, there are log entries in the search results for both `status:error` logs and `status:success` logs. If you extracted lists of unique order IDs from the two queries, this order ID would appear in both. With subqueries, you can obtain a mutually exclusive list.
+
+In this example, you are only interested in logs for orders that did not eventually succeed. To filter out orders that eventually succeeded using the subquery feature:
```

Comment:
```suggestion
In this example, you are only interested in logs for orders that did not eventually succeed. To exclude orders that eventually succeeded using the subquery feature:
```
I had to re-read this a few times to understand the meaning, so I think to the reader, the word "exclude" would make more sense vs "filter out", but kind of your preference :)